cli/create/templates/cartridge/README.md

```diff
@@ -0,0 +1,78 @@
+# Simple Tarantool Cartridge-based application
+
+This a simplest application based on Tarantool Cartridge.
+
+## Quick start
+
+To build application and setup topology:
+
+```bash
+cartridge build
+cartridge start -d
+cartridge replicasets setup --bootstrap-vshard
+```
+
+Now you can visit http://localhost:8081 and see your application's Admin Web UI.
+
+**Note**, that application stateboard is always started by default.
+See [`.cartridge.yml`](./.cartridge.yml) file to change this behavior.
```

Comment:
Maybe add information about

```bash
--stateboard=false
```

flag?

Reply:
It seems logical, by passing `--stateboard=false` on `start` requires passing it on `stop` (otherwise, you will get error).
I think that this little note is enough for the first start of application.